packages/web/src/common/components/Modal/ModalExample.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import Modal from "./Modal";
+
+import Button from "../Button";
+
+const ModalBody = styled.div`
+  display: inline-flex;
+  padding: 32px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+`;
```

Comment:
생각해보니 `Modal.tsx`에도 어느 정도의 default layout style이 있는 것이 좋을 것 같아서.. 이거 그냥 `Modal.tsx`의 `ModalBlockContainer` style에도 넣어주실래요?? 
(피그마에 있는 예시 모달들에서 따온 값들 같은데, gap 10px인거 제가 12px로 막 고쳐서 12px로 부탁드립니당 ㅎㅎㅎ)

```suggestion
const ModalBody = styled.div`
  display: inline-flex;
  padding: 32px;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  gap: 12px;
`;
```

Reply:
요거 gap 스타일 적용해보려고 했는데.. 직접 ModalContainer에 넣으면 gap 적용이 안 되거나 overflow시 스크롤 동작이 안되거나 이상하게 되는 것 같아서 일단 패딩 부분만 안에 넣고 스크롤 컨테이너 따로 만들어두고 나머지 정렬이나 gap은 여전히 밖에 있습니다.. ㅜㅜ 
그리고 스크롤바 때문에 width가 더 늘어나거나 스크롤바가 패딩 안쪽에 위치하고 해서 스크롤 컨테이너를 따로 추가하였습니다..! 혹시 스크롤은 스타일 변경없이 디폴트 스타일로 가는건가요?

Reply:
오 그렇군 그럼 gap은 밖에 있어도 괜찮을 것 같아용
스크롤 디자인도 나중에 추가해야겠군요 일단 후순위로 미룰게요ㅎㅎㅎ 당장은 스크롤 해야할만큼 긴 모달은 없을 것 같긴 합니다!